### PR TITLE
Instructions for git lfs

### PR DESCRIPTION
**Pull request by drewvolz**
We added `git lfs` recently. It requires some setup.

### CODE REVIEW

File: README.md
Comment (hawkrives) [inline README.md]: Nit: Don't expand the acronym here.

File: README.md
Comment (hawkrives) [inline README.md]: Add `git lfs install` first. I managed to not install it system-wide, so let's be on the safe side.

**Review by hawkrives — CHANGES_REQUESTED** [review]

File: README.md
Comment (rye) [reply README.md]: I think it only needs to be run once, but if this is peoples' first time using it, they should run that.  I think that since it adds LFS as a hook, cloning a LFS repository will probably also automatically pull them, but since these two operations are non-destructive, we don't need to worry about them.

Comment (drewvolz) [reply README.md]: That's part of the installation instructions in the link.

File: README.md
Comment (drewvolz) [reply README.md]: We define every term we use. I don't see why not?

File: README.md
Comment (drewvolz) [reply README.md]: @rye are you saying we should not include instructions for this at all?

**Review by rye — APPROVED** [review]
I'll approve, pending @hawkrives' approval.  I'm of the mind that these installation steps should include everything that is needed in order to get started, and I think that a proper installation of LFS will point to this solution.

Anyhow. Otherwise looks good to me.

File: README.md
Comment (hawkrives) [reply README.md]: Can we have the abbreviation here, too, then, please? "Git Large File Storage (LFS)" maybe?

File: README.md
Comment (hawkrives) [reply README.md]: I'm thinking that we could include the os x install instructions inline, since they're `brew install git-lfs` and `git lfs install`

Comment (hawkrives) [reply README.md]: and then link to the installation page for other options

File: README.md
Comment (hawkrives) [inline README.md]: "Install Git Large File Storage (LFS)"

I guess that link's fine. It does more hand-holding than anything else I've found.

File: README.md
Comment (hawkrives) [inline README.md]: I still want `git lfs install` before this. However I found my original instructions, I managed to not get it activated on this repo. It's an idempotent operation, more or less, though, so I'd still like to put it here. Please?

**Review by hawkrives — CHANGES_REQUESTED** [review]
last tweakses

**Review by hawkrives — APPROVED** [review]

File: README.md
Comment (rye) [reply README.md]: @drewvolz No, I would say we should include this step to make sure that people have that.

**Pull request merged**
